nrpy/equations/nrpyelliptic/CommonParams.py

```diff
@@ -0,0 +1,54 @@
+"""
+Common parameters for NRPyElliptic.
+
+Author: Thiago Assumpção
+        assumpcaothiago **at** gmail **dot* com
+
+License: BSD 2-Clause
+"""
+
+# Step P1: Import needed NRPy+ core modules:
+import nrpy.params as par  # NRPy+: Parameter interface
+
+thismodule = __name__
```

Comment:
Probably easiest to just use __name__ below, and not define thismodule.

Reply:
Fixed in https://github.com/nrpy/nrpy/pull/15/commits/eb9df10192db337d3a51204318207e716113b581.